mount-daemon/controllers/clientmount_controller.go

```diff
@@ -128,7 +128,7 @@ func (r *ClientMountReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	if clientMount.Spec.DesiredState == dwsv1alpha2.ClientMountStateMounted {
 		err := r.mountAll(ctx, clientMount)
 		if err != nil {
-			resourceError := dwsv1alpha2.NewResourceError("Mount failed", err)
+			resourceError := dwsv1alpha2.NewResourceError("Mount failed").WithError(err)
```

Comment:
Some of these messages begin with capitalized words, some don't. Consider making them consistent.